hal/src/thumbv7em/sercom/v2/spi.rs

```diff
@@ -804,26 +804,33 @@ impl Length for DynLength {
 /// Marker trait for statically known transaction [`Length`]s
 pub trait StaticLength: Length {}
 
+/// Marker trait for transactions that are performed atomically
+pub trait AtomicLength: Length {}
+
```

Comment:
Ah, I see you already implemented it. That's fine. I'll adjust as necessary in my refactor.